charts/nextcloud/files/nginx.config.tpl

```diff
@@ -3,7 +3,17 @@ upstream php-handler {
 }
 
 server {
+    {{- if and (not (has "IPv4" .Values.nginx.ipFamily)) (has "IPv6" .Values.nginx.ipFamily) }}
+    # Only IPv6 is enabled
+    listen [::]:{{ .Values.nginx.containerPort }};
+    {{- else if and (has "IPv4" .Values.nginx.ipFamily) (has "IPv6" .Values.nginx.ipFamily) }}
+    # Both IPv4 and IPv6 are enabled
+    listen [::]:{{ .Values.nginx.containerPort }};
+    ipv6only = off;
```

Comment:
I'd prefer the order of these to be the other way around, such that 4+6 use-case is first, then 6 only, and 4 only as fallback.